File: problems/python/6.py
#The sum of the squares of the first ten natural numbers is

#1^2 + 2^2 + ... + 10^2 = 385

#The square of the sum of the first ten natural numbers is

#(1 + 2 + ... + 10)^2 = 55^2 = 3025

#Hence the difference between the sum of the squares of the first ten natural numbers and the square of the sum is

#3025 - 385 = 2640

#Find the difference between the sum of the squares of the first one hundred natural numbers and the square of the sum.

NUMBER = 100

def sum_of_squares(n):
    return n * (n + 1) * (2 * n + 1) / 6

def square_of_sum(n):
    return (n * (n + 1) / 2) ** 2

if __name__ == "__main__":
    print(int(abs(sum_of_squares(NUMBER) - square_of_sum(NUMBER))))

File: problems/c/6.c
#include <stdio.h>
#include <stdlib.h>
int limit = 100;

int squareOfSum(int n){
    int sum1=(n*(n+1)*(2*n+1))/6;
    return sum1;
}
int sumOfSquares(int n){
    int sum2=(n*(n+1))/2;
    return sum2*sum2;
}
void main(){
    printf("%d",(abs(squareOfSum(limit)-sumOfSquares(limit))));
}
